django_large_image/rest/data.py

```diff
@@ -35,7 +35,7 @@ def thumbnail(self, request: Request, pk: int = None, fmt: str = 'png') -> HttpR
         encoding = tilesource.format_to_encoding(fmt)
         width = int(self.get_query_param(request, 'max_width', 256))
         height = int(self.get_query_param(request, 'max_height', 256))
-        source = self.get_tile_source(request, pk)
+        source = self.get_tile_source(request, pk, encoding=encoding)
         thumb_data, mime_type = source.getThumbnail(encoding=encoding, width=width, height=height)
```

Comment:
@manthey, should we open an image with the same encoding as we are requesting for the thumbnail? Or should this not matter? What do you recommend?